### PR TITLE
chore: update pr template

### PR DESCRIPTION
**Pull request by lukaisailovic**
# Description
New PR template structure

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 # Associated Issues
 
 closes #...
+
+# Checklist
+
+- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
+- [ ] My changes generate no new warnings
```
Comment (tomiir) [inline .github/pull_request_template.md:20]:
```suggestion
# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [ ] My changes generate no new warnings
- [ ] I have reviewed my own code
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
-# Changes
+## Type of change
 
-- feat:
-- fix:
-- chore:
+- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
```
Comment (tomiir) [inline .github/pull_request_template.md:10]: Do we need this field? We can directly mark the PR as draft for this right?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,13 +1,25 @@
-# Breaking Changes
+# Description
 
-N/A
+Please include a brief summary of the change.
```
Comment (bkrem) [inline .github/pull_request_template.md:3]: Would it make sense to have an optional backlink to the relevant Linear issues here?
```suggestion
Please include a brief summary of the change.

Linear issue: APKT-xxx
```

Comment (enesozturk) [reply .github/pull_request_template.md:3]: +1 Ideally every PR should have been tracked on Linear and we should be getting the branch name from the Linear, so it'll automatically be linked to Linear and the ticket will be moved to next stage when the PR is merged.

**Review by svenvoskamp — APPROVED** [review]

**Review by enesozturk — APPROVED** [review]

**Pull request merged**